pkg/helm/v3/dependency.go

```diff
@@ -17,3 +27,22 @@ func (h *HelmV3) DependencyUpdate(chartPath string) error {
 	}
 	return man.Update()
 }
+
+// garbageCollect walks over the files in the given path and deletes
+// any anonymous index file with a mod time older than the given
+// duration.
+func garbageCollect(path string, olderThan time.Duration) {
+	now := time.Now()
+	filepath.Walk(path, func(p string, f os.FileInfo, err error) error {
+		if err != nil || f.IsDir() {
+			return nil
+		}
+		if strings.HasSuffix(f.Name(), "=-index.yaml") || strings.HasSuffix(f.Name(), "=-charts.txt") {
+			if diff := now.Sub(f.ModTime()); diff > olderThan {
```

Comment:
Could this just be `if now.Sub(f.ModTime()) > olderThan`? (perhaps the diff value was used, in an earlier version?)

Reply:
You are right, some previous version did rely on a `time.Time` I think (and I thus needed to make a transformation to a comparable format iirc).